fcgl/madrid-native
# Implement recent serach history api connection

## Problem

Currently have recent history hard coded, need to make an api call to get the user's recent search history

## Solution

Implement API request

## Test
1. Run search repository
2. Run native app
3. Make a search
4. Confirm that the search now appears under recent searches
